docker/start_server.sh

```diff
@@ -75,49 +18,26 @@ function symlink_client_files() {
     )
 }
 
-function fdb_to_sqlite() {
-    echo "Run fdb_to_sqlite"
-    python3 /app/utils/fdb_to_sqlite.py /client/client/res/CDClient.fdb --sqlite_path /app/res/CDServer.sqlite 
-
-    (
-        cd /app/migrations/cdserver
-        readarray -d '' entries < <(printf '%s\0' *.sql | sort -zV)
-        for entry in "${entries[@]}"; do
-            echo "Execute $entry"
-            sqlite3 /app/res/CDServer.sqlite < $entry
-        done
-    )
+function symlink_config_files() {
+    echo "Creating symlinks for config files"
+    rm /app/*.ini
+    ln -s /configs/authconfig.ini /app/authconfig.ini
+    ln -s /configs/chatconfig.ini /app/chatconfig.ini
+    ln -s /configs/masterconfig.ini /app/masterconfig.ini
+    ln -s /configs/worldconfig.ini /app/worldconfig.ini
 }
 
-set_defaults
-
-check_sql_connection
-
-update_ini_values
-
-if [[ ! -d "/client" ]]; then
-    echo "Client not found."
-    echo "Did you forgot to mount the client into the \"/client\" directory?"
-    exit 1
-fi
-
-if [[ ! -f "/client/extracted" ]]; then
-    echo "Start client resource extraction"
-
-    python3 /app/utils/pkextractor.py /client/ /client/
-
-    touch /client/extracted
+if [[ ! -f "/app/initialized" ]]; then
+    # setup symlinks for volume files
+    symlink_client_files
+    symlink_config_files
+    # do not run symlinks more than once
+    touch /app/initialized
 else
-    echo "Client already extracted. Skip this step"
-    echo "If you want to force re-extract, just delete the file called \"extracted\" in the client directory"
+    echo "Server already initialized"
 fi
 
-symlink_client_files
-
-fdb_to_sqlite
-
+# start the server
```

Comment:
We need to change this here a bit. There a two options:

1. We add a health check to the `setup` container which returns healthy if the setup is complete and then we can mark this container as depends_on with the condition healthy
2. We add a loop here like this:

```bash
while [ ! -f "/client/migrated" ]; do
    echo "Client setup not finished. Waiting for setup container to complete"
    sleep 1
done
```

Reply:
I like #2 the best. This way you don't actually have to run the `setup` container each time you want to launch the main server. Just run it once and the server is ready to go.